cfgov/jinja2/v1/_includes/organisms/image-text-50-50-group.html

```diff
@@ -24,7 +26,7 @@
         <div class="u-mt30">
             {{ social_media.render( {
                 'is_share_view': true,
-                'blurb': share_blurb
+                'blurb': share_blurb,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2775/commits/e294ca8faff63b276006716a305323fe4ff13a30